letsencrypt/cli.py

```diff
@@ -1158,19 +629,16 @@ class HelpfulArgumentParser(object):
 
     """
 
-    # Maps verbs/subcommands to the functions that implement them
-    VERBS = {"auth": obtain_cert, "certonly": obtain_cert,
-             "config_changes": config_changes, "everything": run,
-             "install": install, "plugins": plugins_cmd, "renew": renew,
-             "revoke": revoke, "rollback": rollback, "run": run}
+    def __init__(self, args, plugins, detect_defaults=False):
 
-    # List of topics for which additional help can be provided
-    HELP_TOPICS = ["all", "security",
-                   "paths", "automation", "testing"] + list(six.iterkeys(VERBS))
+        from letsencrypt import main
+        self.VERBS = main.VERBS
+        # List of topics for which additional help can be provided
+        HELP_TOPICS = ["all", "security",
+                       "paths", "automation", "testing"] + list(six.iterkeys(self.VERBS))
 
-    def __init__(self, args, plugins, detect_defaults=False):
         plugin_names = list(six.iterkeys(plugins))
```

Comment:
Same is true of this line `list(plugins)`, but I know using `iterkeys` wasn't introduced in this PR.